tap_twilio/streams.py

```diff
@@ -12,313 +12,125 @@
 #   bookmark_type: Data type for bookmark, integer or datetime
 #   children: A collection of child endpoints (where the endpoint path includes the parent id)
 #   parent: On each of the children, the singular stream name for parent element
-
+# todo review task_channels.json, review all fields with json information that needs extra work, TaskQueue Statistics
 STREAMS = {
     # Reference: https://www.twilio.com/docs/usage/api/account#read-multiple-account-resources
-    'accounts': {
-        'api_url': 'https://api.twilio.com',
-        'api_version': '2010-04-01',
-        'path': 'Accounts.json',
-        'data_key': 'accounts',
+
+    'workspaces': {
+        'api_url': 'https://taskrouter.twilio.com',
+        'api_version': 'v1',
+        'path': 'Workspaces',
+        'data_key': 'workspaces',
         'key_properties': ['sid'],
-        'replication_method': 'FULL_TABLE',  # Fetch ALL, filter results
+        'replication_method': 'FULL_TABLE',
```

Comment:
Stream should use `FULL_TABLE` method only if incremental extract is absolutely not possible. Can we fetch this endpoint incrementally and change it to `INCREMENTAL` by any chance?

There is a `'replication_keys': ['date_updated']` for this stream one line below, but we're not using `INCREMENTAL`. In this case it will not create the state file and every tap run will fetch all the data from the the earliest time possible.